enferno/admin/views.py

```
@@ -6588,3 +6591,513 @@
+            current_user,
+            Activity.ACTION_UPDATE,
+            Activity.STATUS_SUCCESS,
+            {"id": field.id, "name": field.name, "entity_type": field.entity_type},
+            "dynamic_field",
+        )
+
+        return HTTPResponse.success(
+            message=f"Updated dynamic field '{field.name}'", data={"item": field.to_dict()}
+        )
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)